fastapi_poe/src/fastapi_poe/base.py

```diff
@@ -48,6 +62,19 @@ def exception_handler(request: Request, ex: HTTPException):
     logger.error(ex)
 
 
+http_bearer = HTTPBearer()
+
+
+def auth_user(authorization: str = Depends(http_bearer)) -> None:
+    logger.info(f"auth is called key is {auth_key} and credition is {authorization}")
```

Comment:
Probably shouldn't log the auth key for security

Reply:
Yeah. Definitely. Should catch this earlier.